src/alchemtest/amber/simplesolvated/descr.rst

```diff
@@ -0,0 +1,22 @@
+Amber: Small molecule thermodynamic integration free energy difference in water. This example uses ligands 17124-1 to 18637-1 from J. Am. Chem. Soc. 2015, 137, 2695−2703. 
+=========================
+
+Notes
+-----
+Data Set Characteristics:
+    :Number of Legs: 2 (charge, vdw)
+    :Number of Windows: 5 for charge, 12 for vdw
+    :Length of Windows: 1ns
+    :System Size: 5979 atoms
+    :Temperature: 300 K
+    :Pressure: 1 bar
+    :Alchemical Pathway: (charge + vdw) in ligand 1 --> (charge + vdw) in ligand 2, charge and vdw are running in parellel, soft core is used in vdw
+    :Experimental Free Energy difference: N/A 
+
```

Comment:
remove empty line (I know, you copied from the GMX dataset... that line did not serve any purpose there either and I took it out in PR #7)

Reply:
Got it, will change this together with other comments once I get the license info finalized.